src/less/components/panels.less

```diff
@@ -15,6 +15,14 @@
   &-body {
     .font-family-base-regular();
   }
+
+  .control {
```

Comment:
In this case is preferable to be modified in the form control: `src/less/components/forms.less` removing the  margin and padding of the `control--checkbox`.